memory.js

```diff
@@ -1,14 +1,22 @@
 /* eslint-disable camelcase */
+if (!window) var { MessageChannel } = require('worker' + '_threads')
```

Comment:
`if (typeof window === 'undefined')` otherwise this will throw

Reply:
fixed by other means (to support webworkers also)